app/views/projects/_form.html.erb

```diff
@@ -596,7 +596,25 @@ provided by the delivered project's software.
 </div>
       <br>
       <div class="center">
-      <% if ! is_disabled %>
+      <% if is_disabled %>
+        <% if project.show_entry_license? %>
+          This data is available under the
+          <a href="https://creativecommons.org/licenses/by/3.0/us"
```

Comment:
For example, the SPDX spec version 2.1 is CC-BY-3.0. I don't expect this data to be directly pulled into the spdx spec, but that is a useful example that not everyone is currently using the 4.0 version of the Creative Commons license.